OpenRA.Mods.Common/Traits/Turreted.cs

```diff
@@ -131,7 +131,8 @@ public bool FaceTarget(Actor self, Target target)
 				return false;
 
 			var pos = self.CenterPosition;
-			var delta = target.Positions.PositionClosestTo(pos) - pos;
+			var targetPos = attack.Info.AttackTargetCenter ? target.CenterPosition : target.Positions.PositionClosestTo(pos);
```

Comment:
'attack' can be null according to implementation of Turreted - `attack != null && ...`